tools/cli/domainCommands.go

```diff
@@ -854,6 +856,70 @@ func (d *domainCLIImpl) migrationDomainWorkFlowCheck(c *cli.Context) DomainMigra
 	}
 }
 
+func (d *domainCLIImpl) searchAttributesChecker(c *cli.Context) DomainMigrationRow {
+	ctx, cancel := newContext(c)
+	defer cancel()
+
+	// getting user provided search attributes
+	searchAttributes := c.StringSlice(FlagSearchAttribute)
```

Comment:
searchAttributes have the format of "key:type". You need to parse that into `map[string]IndexValueType`. You can use the Unmarshal method to read type into `IndexValueType`, for example:
```
var ivt types.IndexValueType
if err := ivt.Unmarshal([]byte(t)); err != nil {
   ErrorAndExit(/* provide details on which search attribute has unknown type */)
}
```